doc/conseil.sql

```diff
@@ -16,17 +16,21 @@ SET xmloption = content;
 SET client_min_messages = warning;
 SET row_security = off;
 
+CREATE SCHEMA IF NOT EXISTS tezos;
+ALTER ROLE conseiluser SET search_path TO tezos,public;
+ALTER DATABASE "conseil-local" SET search_path TO tezos,public;
```

Comment:
Personal preference, I think "conseil" over "conseil-local" would suffice.

Reply:
`conseil-local` is a name of the db in the `docker-compose`, I could change it in the both places to `conseil`, WDYT?